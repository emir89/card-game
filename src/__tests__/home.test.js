import { render } from "@testing-library/react";
import React from "react";
import Home from '../containers/HomePage';
import renderer from 'react-test-renderer';

describe('Home page', () => {
   test('Shows buttons', () => {
       let {queryByText} = render(<Home />);

       expect(queryByText("2 Players")).toBeTruthy();
       expect(queryByText("3 Players")).toBeTruthy();
       expect(queryByText("4 Players")).toBeTruthy();

   });

   test('It matches snapshot', () => {
       const tree = renderer.create(<Home />).toJSON();

       expect(tree).toMatchSnapshot();
   })
});
