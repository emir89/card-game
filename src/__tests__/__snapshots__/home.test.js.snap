// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page It matches snapshot 1`] = `
<div
  className="CenterWrapperColumn"
>
  <div
    className="NumberOfPlayersText"
  >
    <h2>
      Select number of players
    </h2>
  </div>
  <div
    className="NumberOfPlayersButtons"
  >
    <button
      onClick={[Function]}
    >
      2 Players
    </button>
    <button
      onClick={[Function]}
    >
      3 Players
    </button>
    <button
      onClick={[Function]}
    >
      4 Players
    </button>
  </div>
</div>
`;
